pkg/build/builder/cmd/builder.go

```diff
@@ -50,6 +50,7 @@ func run(builderFactory factoryFunc) {
 				glog.Fatalf("Cannot setup secret file for accessing private repository: %v", err)
 			}
 			if sourceURL != nil {
+				build.Annotations[bld.OriginalSourceURLAnnotationKey] = build.Spec.Source.Git.URI
```

Comment:
no, it never makes it back to the master... it's a temporary spot inside the builder code to pass the original URL